WPMedia/fusion-news-theme-blocks
# TMEDIA-435/navigation_cta_logged_in_2 (WIP)

## Description
Included a navigation  call to action at the top of the page. Includes submenu and format for mobile.

All based on designs found here: https://app.zeplin.io/project/603fa53e2626ed1592e7c0e6/screen/6113e173541a64881b12adea

## Jira Ticket
- [TMEDIA-435](https://arcpublishing.atlassian.net/browse/TMEDIA-435)

## Acceptance Criteria
As a logged in user, I can see My Account CTA (call to action) from the navigation.

When a user is in a logged in state on desktop, they should see My Account in the nav CTA button.
arrow facing down 
little person icon shown 
Text says “Account”

On click, a dropdown appears with Manage your account and Sign out links:

no divider line
Manage your account (link to nowhere is ok) 
log out link (again, link to nowhere is ok) 
on click, carrot is facing arrow facing up on click.

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-435/navigation_cta_logged_in_2`
2. In `Fusion-News-Theme`, checkout POC branch: `git checkout TMEDIA-400-subs-poc`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. In PB Editor, create a page and insert a `Navigation` and `Identity-header-account` blocks, with the following config:
<img width="501" alt="Screen Shot 2021-08-24 at 11 29 56 AM" src="https://user-images.githubusercontent.com/82830431/130654766-e4a6b297-845a-4f73-91a9-4e2d9bdda7bb.png">
<img width="491" alt="Screen Shot 2021-08-24 at 11 29 23 AM" src="https://user-images.githubusercontent.com/82830431/130654769-d9817b51-178b-4f54-afc2-947ca7f94f59.png">
<img width="481" alt="Screen Shot 2021-08-24 at 11 29 32 AM" src="https://user-images.githubusercontent.com/82830431/130654772-3dea7b23-10d0-4463-9f40-23f04d2acfd9.png">
4. Observe the behavior of the button at the top right
5. Click outside of open menu to close

## Effect Of Changes
<img width="212" alt="Screen Shot 2021-08-24 at 10 36 32 AM" src="https://user-images.githubusercontent.com/82830431/130646676-d8bc7f6b-027f-44ae-8a69-9ca31e7b4847.png">
<img width="261" alt="Screen Shot 2021-08-24 at 10 36 45 AM" src="https://user-images.githubusercontent.com/82830431/130646679-639b8762-59ff-4465-94b2-c8d0832b84ff.png">


## Dependencies or Side Effects
`Fusion-News-Theme` must be on the branch `TMEDIA-400-subs-poc`

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
